core/src/main/java/com/oath/oak/Chunk.java

```diff
@@ -621,51 +615,17 @@ private int getLastItemEntryIndex() {
         return entryIndex;
     }
 
+
+    public void freeHandle(int handleIndex) {
```

Comment:
please add documentation - what exactly does this method do?

Reply:
I think its pretty clear from the name of the function.